src/server/server.zig

```diff
@@ -93,17 +94,15 @@ pub const User = struct { // MARK: User
 	const simulationSize = 2*maxSimulationDistance;
 	const simulationMask = simulationSize - 1;
 	conn: *Connection = undefined,
-	player: Entity = .{},
 	timeDifference: utils.TimeDifference = .{},
-	interpolation: utils.GenericInterpolation(3) = undefined,
```

Comment:
This is not at all related to the ECS and doesn't belong in this PR, either do this change in a separate PR first so we can closely look at the consequences, or keep the interpolation (you can make a separate stable pointer and manually copy data over in each update)